src/components/HomeListView.js

```diff
@@ -33,7 +29,7 @@ export class HomeListView extends Component {
         cellHeight={50}
         cellProps={{ onPress: this.onGroupSelect }}
         pageSize={5}
-        sectionHeader={() => null}
+        sectionHeader={nullFn}
```

Comment:
Because a sectionHeader component is supposed to be mandatory to use AlphabetListView correctly... Except we never created it, so we render this "null component". 

I'm fairly sure removing this line would just give us a warning to add the component, which, thinking about it, could be a good idea ^^'